demo/billing/src/randomizer.rs

```diff
@@ -30,10 +29,10 @@ pub struct RecordState {
 }
 
 fn protobuf_timestamp(time: DateTime<Utc>) -> Timestamp {
-    let mut ret = Timestamp::new();
-    ret.seconds = time.timestamp();
-    ret.nanos = time.timestamp_subsec_nanos() as i32;
-    ret
+    Timestamp {
```

Comment:
constructing the structs this way alone makes me like this change a bunch lol

Reply:
I know right?

Reply:
why was it this way before? we didn't have to do this in persist

Reply:
I think this is just how the rust-protobuf docs showed it, so that you didn't have to deal with `unknown_fields`.